app/src/main/java/org/tokend/template/view/util/CopyDataDialogFactory.kt

```diff
@@ -11,7 +11,7 @@ object CopyDataDialogFactory {
 
     fun getDialog(
             context: Context,
-            content: String,
+            content: String?,
```

Comment:
What's the purpose of data copy dialog when there is no data? 🤔 If there is no data then `.getDialog` must not be called